sunbird_mcp/src/config.py

```
@@ -0,0 +1,136 @@
+        env="SUNBIRD_API_ENDPOINT_SEARCH",
+        description="Endpoint for searching content"
+    )
+    
+    API_ENDPOINT_READ: str = Field(
+        default="/api/content/v1/read",
+        env="SUNBIRD_API_ENDPOINT_READ",
+        description="Endpoint for reading content details"
+    )
+    
+    @field_validator('API_BASE_URL')
+    def validate_api_base_url(cls, v):
```

Pylint (3.3.7)

[error] Method 'validate_api_base_url' should have "self" as first argument

(E0213)